sdk/storage/azure-storage-files-shares/test/ut/share_directory_client_test.cpp

```diff
@@ -820,6 +820,34 @@ namespace Azure { namespace Storage { namespace Test {
     }
   }
 
+  TEST_F(FileShareDirectoryClientTest, ListFilesAndDirectoriesEncoded)
```

Comment:
@microzchang where is the fix for the invalid xml? Given the PR title, I expected a source code change or an issue number. This PR looks to just change test code and add a unit test.

Please update the PR title to be more descriptive/accurately reflect what the intent of this change is to avoid confusion :)

Reply:
@ahsonkhan Thanks for your suggestion. The fix for the invalid xml has done in this PR: https://github.com/Azure/azure-sdk-for-cpp/pull/4102 and haven't add unit test yet. This PR is mainly for convenience layer and unit test about **Files Invalid XML**. But it seems like there's nothing else to do with convenience layer so I just added unit test for it.
I will change the PR title.